File: Giovanna-e-Gabriela/CPP/Lista-1/ex-04.cpp
/*
Data: 05/05/2026
Autores: Gabriela Onaga e Giovanna Costa 
Descrição: Ler 10 números. Verificar e exibir o menor e maior número dessa sequência.
/* 

#include <iostream>
using namespace std;

int main() {
    int n, menor, maior;

    for (int i = 1; i <= 10; i++) {
        cout << "Digite um numero: ";
        cin >> n;

        if (i == 1) {
            menor = n;
            maior = n;
        } else {
            if (n < menor) {
                menor = n;
            }
            if (n > maior) {
                maior = n;
            }
        }
    }

    cout << "\nMenor numero: " << menor << endl;
    cout << "Maior numero: " << maior << endl;

    return 0;
}

File: Giovanna-e-Gabriela/Python/python-conversion/python/ex04.py
"""
    Autor: Giovanna Cabral e Gabriela Onaga
    Data: 12/08/2026
    Descrição: Calcula a soma dos números pares de 1 até um número informado.
"""

num = int(input("Digite um numero inteiro positivo: "))
contador = 1
soma = 0

if num > 0:
    while contador <= num:
        if contador % 2 == 0:
            soma = soma + contador
        contador += 1

    print(f"Soma dos pares: {soma}")
else:
    print("Número Inválido.")
